### PR TITLE
samplefilejavamodified

### PR DESCRIPTION
**Pull request by raghu9148617954**
sample pull request

### CODE REVIEW

File: src/test/java/gitsampleproject/Sample.java [hunk 1]
```diff
@@ -7,7 +7,7 @@ public static void main(String[] args)
 	{
 	
 		
-		System.out.println("raghu");
+		System.out.println("raghuKiran");
```
Comment (raghavendra-t) [inline src/test/java/gitsampleproject/Sample.java:10]: it's fine

Comment (raghavendra-t) [reply src/test/java/gitsampleproject/Sample.java:10]: okay

**Review by raghavendra-t — APPROVED** [review]

File: src/test/java/gitsampleproject/Sample.java (hunk 1 above)
Comment (raghu9148617954) [reply src/test/java/gitsampleproject/Sample.java:10]: thanks

**Comment by raghavendra-t** [comment]
All the code was correct

**Pull request merged**